### PR TITLE
Add Android ARCH=arm LLVM_IAS=1 5.4+ coverage

### PR DESCRIPTION
**Pull request by nickdesaulniers**
add thumb2 coverage as well.

### CODE REVIEW

File: generator.yml [hunk 1]
```diff
@@ -263,10 +263,16 @@ builds:
   #############
   #  Android  #
   #############
+  - {<< : *arm32_allmod,      << : *android-mainline, << : *llvm_full,       boot: false, llvm_version: *llvm_tot}
+  - {<< : *arm32_v7_t,        << : *android-mainline, << : *llvm_full,       boot: false, llvm_version: *llvm_tot}
```
Comment (nathanchance) [inline generator.yml:267]: We should be able to boot the Thumb2 ones, right?

Comment (nickdesaulniers) [reply generator.yml:267]: good catch! PTAL

**Review by nathanchance — APPROVED** [review]
LGTM!

**Comment by nathanchance** [comment]
Not sure if you got the email too but there are some new failures: https://github.com/ClangBuiltLinux/continuous-integration2/actions/runs/699686513

**Comment by nickdesaulniers** [comment]
sorry, let me fix those up.

**Comment by nickdesaulniers** [comment]
Strange, I'm having difficulty reproducing the error observed for allmodconfig ARCH=arm (not thumb) w/ LLVM_IAS=1.

example [1](https://github.com/ClangBuiltLinux/continuous-integration2/runs/2223789860?check_suite_focus=true)[2](https://github.com/ClangBuiltLinux/continuous-integration2/runs/2223694315?check_suite_focus=true)[3](https://github.com/ClangBuiltLinux/continuous-integration2/runs/2223640048?check_suite_focus=true).

```
arch/arm/crypto/curve25519-core.S:25:2: error: instruction requires: NEON
 vmov.i32 q0, #1
 ^
```

**Pull request merged**